src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,122 @@
+﻿<UserControl
+    x:Class="Microsoft.Terminal.TerminalControl.SearchBoxControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.Terminal.TerminalControl"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root"
+    HorizontalAlignment="Stretch"
+    VerticalAlignment="Top"
+    d:DesignHeight="75"
+    d:DesignWidth="200"
+    TabNavigation="Cycle">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="{ThemeResource SystemColorButtonTextColor}" />

```

Comment:
```suggestion
                        <Setter Property="Foreground" Value="{ThemeResource SystemColorWindowTextColor}" />
```
In general, background and foreground colors should match as both "Window" or both "Button".  Since you're using SystemColorWindowColor as the Background on line 58, you should match with "Window" here.

Reply:
changed